app/src/main/java/com/sjh/sunflower_sksowk156/feature/main/MainScreen.kt

```diff
@@ -76,10 +75,10 @@ fun MainScreen(modifier: Modifier, onPlantItemClick: (String) -> Unit) {
                 modifier = modifier.fillMaxWidth(),
                 selectedTabIndex = pagerState.currentPage,
             ) {
-                tabTitles.forEachIndexed { index, title ->
-                    val tabImageResource = when (index) {
-                        0 -> R.drawable.ic_my_garden_active
-                        else -> R.drawable.ic_plant_list_active
+                for (index in 0 until pagerState.pageCount) {
+                    val (title, tabImageResource) = when (index) {
+                        0 -> Pair("My garden", R.drawable.ic_my_garden_active)
+                        else -> Pair("Plant list", R.drawable.ic_plant_list_active)
```

Comment:
```suggestion
                        0 -> "My garden" to R.drawable.ic_my_garden_active
                        1 -> "Plant list" to R.drawable.ic_plant_list_active
                        else -> fail("Impossible to reach here")
```
이렇게 해도 아마 될 거에요. `A to B` => `Pair(A, B)`

Reply:
감사합니다 확인해보겠습니다!